docs/concepts/credentials/3_terms-and-quote.md

```diff
@@ -78,30 +73,14 @@ The terms object is sent via the messaging in the SDK using the interface “ISu
 
 ## How to create a quote
 
-A quote is a description of the terms and conditions of the work to be performed. It may be sent to the attester from the claimer, but may also be sent from a claimer to an attester. In cases where multiple attesters provide the same attestation (for example, a car inspection) the claimer may send a quote to several attesters to choose the attester with the best conditions.
+A quote consists of costs, a timeframe and the terms and conditions of the work to be performed. It may be sent to the claimer by the attester. In cases where multiple attesters provide the same attestation (for example, a car inspection) the claimer may request a quote of several attesters to choose the attester with the best conditions.
```

Comment:
Timeframe would be the expiration date?

Reply:
317bd4334a1cbfdb4f1aa7b1bbd7b7e8b7f2dd71

Reply:
Can you maybe add an explanation of what timeframe is supposed to mean?